server/index.js

```diff
@@ -83,28 +85,26 @@ function render(filename, template, res) {
     return;
   }
 
-  var readFile = promisify(fs.readFile);
   var rendr = promisify(res.render.bind(res));
 
   readFile(filename, 'utf8')
     .catch(err => {
-      console.log(err);
-      error(res, 404);
-    }).then(data => {
+      err._code = 404;
+      throw err;
+    }).then(async data => {
       var parts = data.split('---\n');
       var config = yaml.load(parts.shift());
       config.content = md(parts.shift());
       config.cachebust = cachebust;
 
       console.log(config);
 
-      return rendr(template, config).then(str => {
-        htmlCache[key] = str;
-        res.end(str);
-      });
+      var str = await rendr(template, config);
+      htmlCache[key] = str;
+      res.end(str);
     }).catch(err => {
-      console.log(err);
-      error(res, 500);
+      console.error(err);
+      error(err, err._code);
```

Comment:
`error` takes a Request instance as its first argument, not an Error object.

Reply:
Thanks for catching this one! It's fixed now!